### PR TITLE
Update Client details to ant design

### LINKED ISSUE #1117: Update Add remote connection page to use Ant Design
## What needs changed?

Full refactor with Ant Design

## How does it work now?

How does this component work now, and what's wrong with it?

## How should it work?

How do you think this component should work?

## Additional information

Add any other context, files, or screenshots about this request here.

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes
Removing entire page of client details.  Updating admin table to show crucial information and then adding a remove and edit button.

## Related issue
Fixes #1117 

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [x] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
* [x] Tests added (or description of how to test) for any new features.
* [x] User documentation updated for UI or technical changes.

### CODE REVIEW

File: src/main/webapp/resources/js/pages/admin/components/clients/listing/ClientsTables.jsx [hunk 1]
```diff
@@ -17,29 +21,67 @@ import { IconStop } from "../../../../../components/icons/Icons";
 export function ClientsTable() {
   const { updateTable } = useContext(PagedTableContext);
 
+  const removeAndUpdate = async (id) => {
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/admin/components/clients/listing/ClientsTables.jsx:24]: Change to `removeClientAndUpdateTable`

File: src/main/webapp/resources/js/pages/admin/components/clients/listing/ClientsTables.jsx [hunk 2]
```diff
@@ -17,29 +21,67 @@ import { IconStop } from "../../../../../components/icons/Icons";
...
+    await deleteClient(id);
+    updateTable();
+  };
+
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/admin/components/clients/listing/ClientsTables.jsx:28]: Add function docs

File: src/main/webapp/resources/js/pages/admin/components/clients/listing/ClientsTables.jsx [hunk 3]
```diff
@@ -56,36 +98,66 @@ export function ClientsTable() {
...
+      title: i18n("ClientsTable.column.scope"),
+      dataIndex: ["details", "scope"],
+      render(scopes) {
+        const colors = {
+          read: "cyan",
+          write: "geekblue",
+        };
+        return (
+          <div>
+            {scopes.map((g) => (
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/admin/components/clients/listing/ClientsTables.jsx:113]: You're going to hate me but change `g` to `scope` :)

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]
Looks good. Just a few changes

File: src/main/webapp/resources/js/pages/admin/components/clients/listing/ClientsTables.jsx (hunk 3 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/admin/components/clients/listing/ClientsTables.jsx:113]: Ok

File: src/main/webapp/resources/js/pages/admin/components/clients/listing/ClientsTables.jsx (hunk 1 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/admin/components/clients/listing/ClientsTables.jsx:24]: Updated

File: src/main/java/ca/corefacility/bioinformatics/irida/ria/web/ajax/ClientsAjaxController.java [hunk 4]
```diff
@@ -84,13 +85,54 @@ public void deleteClientTokens(@RequestParam Long id) {
...
+	 * @return Client id if success or an error message if there was an error during the update
+	 */
+	@PutMapping
+	@PreAuthorize("hasRole('ROLE_ADMIN')")
+	public ResponseEntity<AjaxResponse> updateClient(@RequestBody CreateUpdateClientDetails request, Locale locale) {
+		try {
+			return ResponseEntity.ok(new AjaxCreateItemSuccessResponse(service.createOrUpdateClient(request)));
+		} catch (Exception exception) {
+			return ResponseEntity.status(HttpStatus.CONFLICT)
+					.body(new AjaxErrorResponse(messageSource.getMessage("server.AddClientForm.error",
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/java/ca/corefacility/bioinformatics/irida/ria/web/ajax/ClientsAjaxController.java]: Updated in d295f24

File: src/main/webapp/resources/js/pages/admin/components/clients/listing/ClientsTables.jsx (hunk 2 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/admin/components/clients/listing/ClientsTables.jsx:28]: Updated

**Review by deepsidhu85 — APPROVED** [review]
Thanks for getting the changes in. Looks great! Great job!

**Pull request merged**